### ISSUE TITLE
UI bug: You get the ability to delete the only member from your org

### ISSUE THREAD

**Issue by tinybigideas** [argoapp-live/beta-testing#20]
## Describe the bug

Selecting the only member in your organisation gives you the option to delete. I haven't clicked through to delete (in case it actually does) but shouldn't offer in the first place.

## Criticality

Medium

## To Reproduce

1. Go to https://aqua.spheron.network/#/members
2. Only have one member
3. Select only member

## Expected behavior

You cannot delete the only member of an organisation. The row trash icon is disabled seemingly for that reason, but the select to delete action is not disabled.

## Screenshots

![image](https://user-images.githubusercontent.com/2805752/164912372-33b920d4-a10b-4be0-aef9-6f278654bc4b.png)

## Desktop (please complete the following information):

 - OS: Windows
 - Browser: Brave

## Your Details

- Web3 Name: tinybigideas
- Twitter handle: tinybigideas

**Comment by rekpero** [comment]
Good catch, it's a critical bug, if someone removes himself from the org, the org will be lost forever.
Need to add a restriction to stop the user from removing himself from the org.

Thanks @tinybigideas for reporting this
